corehq/apps/app_manager/decorators.py

```diff
@@ -51,4 +51,4 @@ def _no_conflict(*args, **kwargs):
     return _no_conflict
 
 require_can_edit_apps = require_permission(Permissions.edit_apps)
-require_deploy_apps = login_and_domain_required  # todo: can fix this when it is better supported
+require_deploy_apps = require_can_edit_apps
```

Comment:
I'm not 100% sure this is right. There are people who will want to install apps on phones but should not be able to edit apps. How is this used exactly?